aviary/validation_cases/benchmark_tests/test_FLOPS_based_sizing_N3CC.py

```diff
@@ -274,7 +275,7 @@ def run_trajectory(sim=True):
         promotes_outputs=['mission:*'])
 
     traj.link_phases(["climb", "cruise", "descent"], [
-                     "time", Dynamic.Mission.ALTITUDE, Dynamic.Mission.MASS, Dynamic.Mission.DISTANCE], connected=strong_couple)
```

Comment:
why was altitude removed?

Reply:
altitude is not a design variable here but is a constraint. prior versions of OM warned about this, recent ones throw an error.